src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -218,6 +218,12 @@ class ChargeFeeFine extends React.Component {
     });
   }
 
+  goBack = () => {
+    const { history } = this.props;
+
+    history.goBack();
```

Comment:
See [comment](https://github.com/folio-org/ui-users/pull/1676#pullrequestreview-632384632) from @zburke in one of the PRs regarding goBack().

Reply:
Thanks for review, @ViktorSoroka07. But looks like here we can not just replace `history.goBack()` with `history.push(path)`, because there 3 places from where we can go to `Create fee/fine page`, and we need to bring user back to prev screen. As the solution, here we can:
- to leave `history,goBack()`
- to ask PO (Holly) if we should change acceptance criteria, and navigate to the one particular page (for example, `Fee/fines history`)  
 @zburke, what do you think?